.github/workflows/redirect.yml

```diff
@@ -1,49 +0,0 @@
-on:
-  push:
-    branches:
-      - main
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
-permissions:
-  contents: read
-  pages: write
-  id-token: write
-# Allow one concurrent deployment
-concurrency:
-  group: "pages"
-  cancel-in-progress: true
-jobs:
-  # this is a bit convoluted but the build and deploy steps had to be
-  # separated as the actions/upload-pages-artifact requires GNU tar to be
-  # present which is not there in the redocusaurus container.
-  build:
-    runs-on: ubuntu-latest
-    container: node:lts-alpine3.17
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Redoc build
-        working-directory: ./docs
-        run: |
-          npm install
-          npm run build
-      - uses: actions/upload-artifact@v3
-        with:
-          name: redocusaurus
-          path: docs/build/
-  deploy:
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/download-artifact@v3
-        with:
-          name: redocusaurus
-          path: public/
-      - name: Setup Pages
-        uses: actions/configure-pages@v3
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
-        with:
-          path: public/
-      - name: Deploy to GitHub Pages
```

Comment:
Are you sure we don't need deploy to pages?

Reply:
Ah ok as we just have the static `index.html` this seems to work. But the links in the html look a bit off to me:

```bash
$ curl https://docs.nineapis.ch
<!DOCTYPE html>
<html>
  <head>
    <title>Nine API docs</title>
    <link rel="icon" type="image/x-icon" href="favicon.ico">
    <meta http-equiv="Refresh" content="0; url=//docs.nine.ch/api" />
  </head>
  <body>
    <p>API docs have moved to <a href="//docs.nine.ch/api">docs.nine.ch</a>.</p>
  </body>
</html>
```

Reply:
You mean the "//docs.nine.ch/api" ref? That will just use whatever protocol was used when hitting docs.nineapis.ch. But you are right. We should always use https.

Reply:
Ah, did not know that would work. As we have a redirect to https anyways it should not matter.